### PR TITLE
Update lab_meetings.md

### PR DESCRIPTION
**Pull request by EleGdt**
Update lab meeting section

## Summary by Sourcery

Update the lab_meetings.md documentation to include comprehensive information about different types of lab meetings, their schedules, and relevant resources.

Documentation:
- Add detailed descriptions of various lab meetings, including regular lab meetings, journal club, one slide meeting, and method meeting, along with relevant links and schedules.

### CODE REVIEW

File: doc/lab_meetings.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Lab meetings (person in charge: Eleonore Giraudet)
```
Comment (marcobarilari) [inline doc/lab_meetings.md:1]:
```suggestion
# Lab meetings (person in charge: Eleonore Giraudet)

```

File: doc/lab_meetings.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Lab meetings (person in charge: Eleonore Giraudet)
+# Regular lab meeting
+At the moment, they happen every thursday at 10:30. Here you can find the  [calendar](https://docs.google.com/spreadsheets/d/1aIu8WcYoRpC9p3fIiilZWTgR7i03lulYj8B5ZyUW_Qc/edit?gid=6486503#gid=6486503) and [here](https://drive.google.com/drive/folders/1Pb75jpul71mMXiP4fSZGEwK34V5meg12) the main gdrive folder where we archive relevant files.
+
+# Journal club
+They happen once a month, here you can propose and/or pick up a paper to be discussed.
+
+# One slide meeting
```
Comment (marcobarilari) [inline doc/lab_meetings.md:8]:
```suggestion
## One slide meeting

```

File: doc/lab_meetings.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+At the moment, they happen every thursday at 10:30. Here you can find the  [calendar](https://docs.google.com/spreadsheets/d/1aIu8WcYoRpC9p3fIiilZWTgR7i03lulYj8B5ZyUW_Qc/edit?gid=6486503#gid=6486503) and [here](https://drive.google.com/drive/folders/1Pb75jpul71mMXiP4fSZGEwK34V5meg12) the main gdrive folder where we archive relevant files.
+
+# Journal club
+They happen once a month, here you can propose and/or pick up a paper to be discussed.
+
+# One slide meeting
+The one slide meeting is once a month, the lab is divided in 3 batches [see here] and 1 batch will present alternatively.
+Each person is invited to present for ~10 minutes 1 slide and the main goal is to update the lab about his/her scientific/project progress.
+
+# Method meeting
```
Comment (marcobarilari) [inline doc/lab_meetings.md:12]:
```suggestion
## Method meeting

```

**Comment by marcobarilari** [comment]
Hey @EleGdt, let me know if you are still in for this and need any help.

**Comment by EleGdt** [comment]
Ok thqnk you for the review, can you check now?

**Review by marcobarilari — APPROVED** [review]

**Pull request merged**